packages/discord.js/src/client/actions/ThreadCreate.js

```diff
@@ -13,6 +13,7 @@ class ThreadCreateAction extends Action {
        * Emitted whenever a thread is created or when the client user is added to a thread.
        * @event Client#threadCreate
        * @param {ThreadChannel} thread The thread that was created
+       * @param {boolean} newlyCreated Whether or not the thread was created or not
```

Comment:
```suggestion
       * @param {boolean} newlyCreated Whether or not the thread was newly created
```

Reply:
Sorry but this is less accurate than the previous suggestion.